lizmap/modules/view/controllers/default.classic.php

```diff
@@ -81,15 +81,17 @@ public function index()
             }
         }
 
-        $title = jLocale::get('view~default.repository.list.title').' - '.$services->appName;
+        $title = $services->appName;
+        $subTitle = jLocale::get('view~default.home.title');
 
         if ($repository) {
             $lrep = lizmap::getRepository($repository);
-            $title = $lrep->getLabel().' - '.$title;
+            $subTitle = $lrep->getLabel().' - '.jLocale::get('view~default.repository.list.title');
         }
-        $rep->title = $title;
+        $rep->title = $title.' - '.$subTitle;
```

Comment:
In case of a project, you will then have "Lizmap - my project blabla". 
Because of how browsers show bookmarks or window title in tabs (as tabs can be tiny), I prefer to have "my project blabla - Lizmap". If a user open several project in his browser, and if he has already several tabs, he could see "Lizmap, m", "Lizmap, f", etc, so it is difficult to find the right tab.

Reply:
To be clear, the title of the window was better before this change. Probably you should not duplicate the title, and you should have a window title different from the page title.

Reply:
I have update the window title, it will be `Landing Page - Application name` or ` Repository title - Projects - Application name`